apps/OboeTester/app/src/main/java/com/mobileer/oboetester/TestOutputActivity.java

```diff
@@ -105,9 +103,17 @@ private void configureChannelBoxes(int channelCount) {
 
     public void closeAudio() {
         configureChannelBoxes(0);
+        mNativeApiSpinner.setEnabled(true);
         super.closeAudio();
     }
 
+    public void startAudio() throws IOException {
+        super.startAudio();
+        int channelCount = mAudioOutTester.getCurrentAudioStream().getChannelCount();
+        configureChannelBoxes(channelCount);
+        mNativeApiSpinner.setEnabled(false);
```

Comment:
mNativeApiSpinner is badly named, by me, so this fix is confusing.
Please rename to mOutputSignalSpinner or something better.
Also rename NativeApiSpinnerListener.

Reply:
And in pauseAudio().